engine/plugins/logger/logger.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * @file
+ * This is a wrapper class to handel the system logger.
+ */
+
+'use strict';
+
+const Logstash = require('logstash-client');
+
+// Static logstash client across loggers.
+let logstashClient = null;
+
+/**
+ * Logger constructor.
+ *
+ * @param host
+ *   Logstash host
+ * @param port
+ *   Logstash port
+ *
+ * @constructor
+ */
+const Logger = function Logger(host, port) {
+    if (logstashClient === null) {
+        logstashClient = new Logstash({
+            type: 'tcp',
+            host: host,
+            port: port
+        });
+    }
+};
+
+/**
+ * Send message to log-stash.
+ *
+ * @param {string} level
+ *   Log level for the message.
+ * @param {string} message
+ *   The message to send to the logger.
+ */
+Logger.prototype.send = function send(level, message) {
+    if (logstashClient !== null) {
+        // This is for to support legacy log messages.
```

Comment:
"This is for to support legacy log messages." => "This is added to support legacy log messages."

Reply:
Fixed